packages/js/elementfinder.js

```diff
@@ -1301,7 +1301,10 @@ class ElementFinder {
             'innertext': [
                 function(elems, input) {
                     return elems.filter(function(elem) {
-                        return (elem.innerText || '').indexOf(input) != -1;
+                        // SVG elements have undefined .innerText, so we fall back to
+                        // .textContent in that case
+                        let text = elem.innerText ?? elem.textContent;
```

Comment:
At first I struggled to see how it could be a problem, but now that you're asking, it might be just my short imagination. Someone may drive an ancient IE with selenium?... Nevertheless, I looked around in the code and found another reason to change it, i.e., style consistency. I've added a new commit (from my work laptop, hence the different username).